geomesa-accumulo/geomesa-accumulo-compute/src/main/scala/org/locationtech/geomesa/compute/spark/GeoMesaSpark.scala

```diff
@@ -88,6 +88,8 @@ object GeoMesaSpark extends LazyLogging {
           useMock: Boolean = false,
           numberOfSplits: Option[Int] = None): RDD[SimpleFeature] = {
     val ds = DataStoreFinder.getDataStore(dsParams).asInstanceOf[AccumuloDataStore]
+    val username = AccumuloDataStoreParams.userParam.lookUp(dsParams).toString
```

Comment:
concerned about seeing changes in this file as the methods were copy/pasted into the AccumuloSpatialRDDProvider, this functionality doesn't appear to be in the execution path for this PR.